libraries/elixir/README.md

```diff
@@ -0,0 +1,58 @@
+Elixir library for Standard Webhooks
+
+Need to handle inbound webhooks in Elixir? Check out [webhoox](https://github.com/cpursley/webhoox).
```

Comment:
I think this reads too much like an ad. Is there a more subtle way of doing it, e.g. "this library was based on code from [webhoox]..."?